src/api/config/environments/test.rb

```diff
@@ -23,7 +23,7 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
-  config.cache_store = :dalli_store, '127.0.0.1:11211', {namespace: ENV['CACHENAMESPACE'], expires_in: 1.hour }
+  config.cache_store = :memory_store
```

Comment:
Does that mean that we previously stored forbidden_projects and alike for a maximum of 1 hour and store them now forever?